schema/spec.md

```diff
@@ -307,13 +409,13 @@ the core xRegistry Resource
   - MUST be a Resource level attribute defined within the `metaattributes`
     section of the model.
 
-#### `format`
+#### 4.2.2. `format`
 
 - Type: String
 - Description: Identifies the schema format. In absence of formal media-type
   definitions for several important schema formats, we define a convention here
   to reference schema formats by name and version as `<NAME>/<VERSION>`. This
-  specification defines a set of common [schema format names](#schema-formats)
+  specification defines a set of common [schema format names](#43-schema-formats)
```

Comment:
While being able to point users to a section # is handy, one downside to having numbers in section titles is that if we ever renumber things ALL docs (especially ones we don't control) will now have bad links - even if the section in question was just moved. Something to think about.